src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -26,4 +26,16 @@
     public List<Movie> getAll() {
         return movieDao.getAll();
     }
+
+    @Override
+    public Boolean update(Movie movie) {
+        Movie currentMovie = get(movie.getId());
+        movieDao.update(movie);
+        return currentMovie.equals(get(movie.getId()));
```

Comment:
The equality check here might not be the best way to determine if the update was successful. Consider returning the updated entity or void.